diff --git a/node_modules/changeset-recover/src/changesets.js b/node_modules/changeset-recover/src/changesets.js
index a09450b..22e3bcb 100644
--- a/node_modules/changeset-recover/src/changesets.js
+++ b/node_modules/changeset-recover/src/changesets.js
@@ -27,6 +27,8 @@ export async function writeChangeset(change, cwd = process.cwd()) {
       ` by ${authorLinks}`;
   }
 
+  console.log(change.commit, { pr: change.pr?.title, num: change.pr?.number, authorLinks, message });
+
   let text =
     `---\n` +
     `# each of these should be one of "patch", "minor", "major"\n` +
