diff --git a/node_modules/changeset-recover/src/bin.js b/node_modules/changeset-recover/src/bin.js
index c2c5cdf..a677e90 100755
--- a/node_modules/changeset-recover/src/bin.js
+++ b/node_modules/changeset-recover/src/bin.js
@@ -49,7 +49,8 @@ yargs(hideBin(process.argv))
         args.base,
         args.main,
         args.path,
-        args.limit
+        args.limit,
+        args.owner,
       );
       let changesets = await getChangesetList(args.path);
       let untrackedChanges = omitTrackedChanges(changes, changesets);
diff --git a/node_modules/changeset-recover/src/changesets.js b/node_modules/changeset-recover/src/changesets.js
index a09450b..a85389c 100644
--- a/node_modules/changeset-recover/src/changesets.js
+++ b/node_modules/changeset-recover/src/changesets.js
@@ -27,6 +27,8 @@ export async function writeChangeset(change, cwd = process.cwd()) {
       ` by ${authorLinks}`;
   }
 
+  console.log(change.commit, { pr: change.pr?.title, num: change.pr?.number, message });
+
   let text =
     `---\n` +
     `# each of these should be one of "patch", "minor", "major"\n` +
diff --git a/node_modules/changeset-recover/src/github.js b/node_modules/changeset-recover/src/github.js
index 8d9e723..54ae70a 100644
--- a/node_modules/changeset-recover/src/github.js
+++ b/node_modules/changeset-recover/src/github.js
@@ -6,11 +6,8 @@ let octokit = new Octokit({ auth: process.env['GITHUB_TOKEN'] });
 export async function getMergedPRs(cwd = process.cwd(), owner = null) {
   let { org, repo } = await getOwner(cwd);
 
-  if (!org || !repo) {
-    return [];
-  }
-
   if (owner) {
+    console.debug(`Repo: ${org}/${repo} overridden with ${owner}/${repo}`);
     org = owner;
   }
 
@@ -22,6 +19,8 @@ export async function getMergedPRs(cwd = process.cwd(), owner = null) {
       '&per_page=200'
   );
 
+  console.log('get PRs', response);
+
   return response.data;
 }
 
@@ -31,11 +30,8 @@ export async function getMergedPRs(cwd = process.cwd(), owner = null) {
 export async function getCommits(pr, owner = null, cwd = process.cwd()) {
   let { org, repo } = await getOwner(cwd);
 
-  if (!org || !repo) {
-    return [];
-  }
-
   if (owner) {
+    console.debug(`Repo: ${org}/${repo} overridden with ${owner}/${repo}`);
     org = owner;
   }
 
@@ -59,7 +55,7 @@ export function extractPRNumberFromCommitMessage(message) {
 
 /**
  * @param {string} cwd current working directory, defaults to process.cwd()
- * @returns {Promise<{ org?: string | undefined; repo?: string | undefined}>}
+ * @returns {Promise<{ org: string; repo: string | undefined}>}
  */
 export async function getOwner(cwd = process.cwd()) {
   // ❯ git config --get remote.origin.url
@@ -68,15 +64,26 @@ export async function getOwner(cwd = process.cwd()) {
     cwd,
   });
 
-  let match = /github\.com(:|\/)(?<org>[^/]+)\/(?<repo>[^.]+)\.git/.exec(
+  // 3 Formats
+  //   https://github.com/NullVoxPopuli/embroider
+  //   https://github.com/NullVoxPopuli/embroider.git
+  //   git@github.com:NullVoxPopuli/embroider.git
+  let match = /github\.com(:|\/)(?<org>[^/]+)\/(?<repo>[^.]+)(\.git)?/.exec(
     stdout
   );
 
+  console.log({ match, stdout });
+
   if (!match) {
-    return {};
+    throw new Error(`URL was not recognized. Received: ${stdout}`);
   }
 
   let { org, repo } = match.groups || {};
 
+  if (!org || !repo) {
+    throw new Error(`Regex failure: org (${org}) or repo (${repo}) were not detected from the URL: ${stdout}`);
+
+  }
+
   return { org, repo };
 }
diff --git a/node_modules/changeset-recover/src/workspaces.js b/node_modules/changeset-recover/src/workspaces.js
index d058cc8..c240f64 100644
--- a/node_modules/changeset-recover/src/workspaces.js
+++ b/node_modules/changeset-recover/src/workspaces.js
@@ -80,7 +80,7 @@ export async function getGroupedChanges(
 
   console.debug('Calculating changes for ref: ' + tag);
 
-  let prs = await getMergedPRs(cwd);
+  let prs = await getMergedPRs(cwd, owner);
   let commits = await mergesToBranch(tag, branch, cwd);
   let projects = await getProjects(cwd);
 
